src/ServiceControl.Audit.AcceptanceTests/Auditing/FailedAuditsController.cs

```diff
@@ -6,35 +6,27 @@
     using System.Threading.Tasks;
     using System.Web.Http;
     using Audit.Auditing;
-    using Infrastructure.WebApi;
-    using Raven.Client;
-    using ServiceControl.Audit.Persistence.RavenDb.Indexes;
+    using ServiceControl.Audit.Infrastructure.WebApi;
+    using ServiceControl.Audit.Persistence;
 
     class FailedAuditsController : ApiController
     {
-        public FailedAuditsController(IDocumentStore store, ImportFailedAudits failedAuditIngestion)
+        public FailedAuditsController(ImportFailedAudits failedAuditIngestion, IFailedAuditStorage failedAuditStorage)
         {
-            this.store = store;
             this.failedAuditIngestion = failedAuditIngestion;
+            this.failedAuditStorage = failedAuditStorage;
         }
 
         [Route("failedaudits/count")]
         [HttpGet]
         public async Task<HttpResponseMessage> GetFailedAuditsCount()
         {
-            using (var session = store.OpenAsyncSession())
-            {
-                var query =
-                    session.Query<FailedAuditImport, FailedAuditImportIndex>().Statistics(out var stats);
-
-                var count = await query.CountAsync();
+            var count = await failedAuditStorage.GetFailedAuditsCount();
 
-                return Request.CreateResponse(HttpStatusCode.OK, new FailedAuditsCountReponse
-                {
-                    Count = count
-                })
-                    .WithEtag(stats.IndexEtag.ToString());
-            }
+            return Request.CreateResponse(HttpStatusCode.OK, new FailedAuditsCountReponse
+            {
+                Count = count
+            }).WithEtag("");
```

Comment:
Do we even make assertions about this etag? If not, we can probably just drop it.

Reply:
Agreed, will fix on the raven branch